Code/Tools/SceneAPI/SceneCore/DataTypes/Rules/ISkinRule.h

```diff
@@ -17,6 +18,36 @@ namespace AZ
     {
         namespace DataTypes
         {
+            inline static constexpr AZStd::string_view DefaultMaxSkinInfluencesPerVertexKey{ "/O3DE/SceneAPI/SkinRule/DefaultMaxSkinInfluencesPerVertex" };
+            inline static constexpr AZStd::string_view DefaultWeightThresholdKey{ "/O3DE/SceneAPI/SkinRule/DefaultWeightThreshold" };
+
+            struct SkinRuleSettings
+            {
+                AZ::u32 m_maxInfluencesPerVertex = 8;
+                float m_weightThreshold = 0.001;
+            };
+
+            inline SkinRuleSettings GetDefaultSkinRuleSettings()
+            {
+                SkinRuleSettings defaultSettings;
+                // Get the project wide default settings for the skin rule
+                if (auto settingsRegistry = AZ::SettingsRegistry::Get(); settingsRegistry != nullptr)
+                {
+                    AZ::u64 defaultMaxSkinInfluences = 0;
+                    if (settingsRegistry->Get(defaultMaxSkinInfluences, SceneAPI::DataTypes::DefaultMaxSkinInfluencesPerVertexKey))
+                    {
+                        defaultSettings.m_maxInfluencesPerVertex = aznumeric_caster(defaultMaxSkinInfluences);
+                    }
+
+                    double defaultWeightThreshold = 0.0f;
+                    if (settingsRegistry->Get(defaultWeightThreshold, SceneAPI::DataTypes::DefaultWeightThresholdKey))
+                    {
+                        defaultSettings.m_weightThreshold = aznumeric_caster(defaultWeightThreshold);
+                    }
+                }
+                return defaultSettings;
+            }
+
```

Comment:
nit Ideally these would be moved to a cpp file instead of being inline in the interface file.